tests/integration/test_integ.py

```diff
@@ -0,0 +1,129 @@
+# HOW TO EXECUTE?
+# First install the package: pip install pytest-docker-compose
+# Run with: pytest --docker-compose="path_to_docker_compose_yml" test_integ.py
+
+import pytest
+import os
+import subprocess
+import unittest
+import time
+import sys
+import logging
+
+
+class TestIntegratedTool(unittest.TestCase):
+
+
+    def setUp(self):
+        print("/************************************* Entered setUp *************************************/")
+
+        self.mounted_files = []
+        self.FUSE_MOUNTED_FILES_LIST = "/tmp/Mounted_files.lst"
+        self.MDH_CORE = "core-sfs"
+        self.MOUNTED_FROM = "/home/sfsuser/Vaidehi/amos-rep/test_tree"
+
+
+        # Check if mdh core is running
+        is_core_up = subprocess.Popen(["docker", "container", "inspect", "-f",
+	                              "'{{.State.Running}}'", self.MDH_CORE], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = is_core_up.communicate()
+        if "true" not in out.decode():
+            print("MDH CORE NOT RUNNING!!!")
+            exit()
+
+        docker_compose_up=subprocess.Popen(["docker-compose", "up"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        docker_up_out, docker_up_err = docker_compose_up.communicate()
+        if docker_up_err.decode().__contains__("error") == True:
+            print("ERROR RUNNING DOCKER-COMPOSE UP!")
+            exit()
+
+        # Check if subprocess is running
+        is_up=docker_compose_up.poll()
+
+        # Returns 0 if subprocess is alive, otherwise 1
+        if is_up is not None:
+            docker_start = subprocess.Popen(
+                ["docker", "container", "start", "synthetic-file-system"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
with the recommended docker library you can simply put `client.containers.get` to get the container, it would raise e.g. a `docker.errors.NotFound` exception